File: scripts/scala/akka/src/main/scala/com/github/diegopacheco/sandbox/scala/akka/remote/actors/HelloWorldActor.scala
package com.github.diegopacheco.sandbox.scala.akka.remote.actors

import akka.actor.Actor._
import akka.actor._

class HelloWorldActor extends Actor {
  def receive = {
    case "Hello" =>
      println("[server] Received Hello, sending: World")
      self.reply("World")
  }
}

File: scripts/scala/akka/src/main/java/com/github/diegopacheco/sandbox/scala/akka/java/actors/HelloWorldActor.java
package com.github.diegopacheco.sandbox.scala.akka.java.actors;

import akka.actor.UntypedActor;

class HelloWorldActor extends UntypedActor {
	public void onReceive(Object msg) {
		getContext().tryReply(msg + " World");
	}
}